web/text_layer_builder.js

```diff
@@ -350,7 +357,9 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
       });
       div.addEventListener('mouseup', function (e) {
         if (self.enhanceTextSelection && self.textLayerRenderTask) {
-          self.textLayerRenderTask.expandTextDivs(false);
+          expandDivsTimer = setTimeout(function() {
+            self.textLayerRenderTask.expandTextDivs(false);
+          }, EXPAND_DIVS_TIMEOUT);
```

Comment:
Similar to comment https://github.com/mozilla/pdf.js/pull/7618/files#r78131484, let's do:
```javascript
//#if !(MOZCENTRAL || FIREFOX)
          expandDivsTimer = setTimeout(function() {
            self.textLayerRenderTask.expandTextDivs(false);
            expandDivsTimer = null;
          }, EXPAND_DIVS_TIMEOUT);
//#else
//        self.textLayerRenderTask.expandTextDivs(false);
//#endif
```